v63004/outref/gtm8922.txt

```diff
@@ -0,0 +1,341 @@
+# Testing VIEW <KEYWORD>[:<region-list>]
+# Create a 3 region DB with gbl_dir mumps.gld and regions DEFAULT, AREG, and BREG
+# Run generalTest to test all commands of the form VIEW <KEYWORD>[:<region-list>]
+# Tests for:
+#    	-VIEW commands accepting region sub-argument accept comma (,) delimited region lists
+# 	-GT.M sorts the regions, eliminating any duplicates from the list. 
+TESTING KEYWORD: DBFLUSH
+------------------------
+n_dsk_write Buffer (AREG): 0x0000000000000000
+n_dsk_write Buffer (BREG): 0x0000000000000000
+n_dsk_write Buffer (DEFAULT): 0x0000000000000000
+VIEW "DBFLUSH":"AREG,BREG,BREG"
+n_dsk_write Buffer (AREG): 0x0000000000000004
+n_dsk_write Buffer (BREG): 0x0000000000000004
+n_dsk_write Buffer (DEFAULT): 0x0000000000000000
+VIEW "DBFLUSH"
+n_dsk_write Buffer (AREG): 0x0000000000000004
+n_dsk_write Buffer (BREG): 0x0000000000000004
+n_dsk_write Buffer (DEFAULT): 0x0000000000000004
+
+
+TESTING KEYWORD: DBSYNC
+------------------------
+n_db_fsync Buffer (AREG): 0x0000000000000000
+n_db_fsync Buffer (BREG): 0x0000000000000000
+n_db_fsync Buffer (DEFAULT): 0x0000000000000000
+VIEW "DBSYNC":"AREG,BREG,BREG"
+n_db_fsync Buffer (AREG): 0x0000000000000001
+n_db_fsync Buffer (BREG): 0x0000000000000001
+n_db_fsync Buffer (DEFAULT): 0x0000000000000000
+VIEW "DBSYNC"
+n_db_fsync Buffer (AREG): 0x0000000000000002
+n_db_fsync Buffer (BREG): 0x0000000000000002
+n_db_fsync Buffer (DEFAULT): 0x0000000000000001
+
+
+TESTING KEYWORD: JNLFLUSH
+------------------------
+n_jnl_flush Buffer (AREG): 0x0000000000000000
+n_jnl_flush Buffer (BREG): 0x0000000000000000
+n_jnl_flush Buffer (DEFAULT): 0x0000000000000000
+VIEW "JNLFLUSH":"AREG,BREG,BREG"
+n_jnl_flush Buffer (AREG): 0x0000000000000001
+n_jnl_flush Buffer (BREG): 0x0000000000000001
+n_jnl_flush Buffer (DEFAULT): 0x0000000000000000
+VIEW "JNLFLUSH"
+n_jnl_flush Buffer (AREG): 0x0000000000000001
+n_jnl_flush Buffer (BREG): 0x0000000000000001
+n_jnl_flush Buffer (DEFAULT): 0x0000000000000001
+
+
+##SUSPEND_OUTPUT JNL_NOBEFORE
+TESTING KEYWORD: JNLFLUSH
```

Comment:
Why are there 2 "Testing JNLFLUSH" section headers in the reference file? If we need to test two statistic counters are updated as part of VIEW "JNLFLUSH", it would be better to check both of them after one VIEW "JNLFLUSH" instead of doing a VIEW "JNLFLUSH" separately for each of the counters.